api/v1beta1/cloudstackaffinitygroup_types.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	AffinityGroupFinalizer = "cloudstackcluster.infrastructure.cluster.x-k8s.io"
```

Comment:
While I don't believe the actual value of this string matters functionally, I'd rather it reflected the CRD it's used with (rather than cloudstackcluster).

Reply:
100% will fix. : )